src/AzureADB2C/Provider.php

```diff
@@ -186,7 +186,7 @@ private function validateIdToken($idToken)
             }
 
             // signature validation and return claims
-            return (array) JWT::decode($idToken, JWK::parseKeySet($this->getJWTKeys(), $this->getConfig('default_algorithm')), $this->getOpenIdConfiguration()->id_token_signing_alg_values_supported);
+            return (array) JWT::decode($idToken, JWK::parseKeySet($this->getJWTKeys(), $this->getConfig('default_algorithm')));
```

Comment:
Can you link to the change as to why the supported algs were removed?

Reply:
Sure, it was a breaking change of version 6.0.0: https://github.com/firebase/php-jwt/releases/tag/v6.0.0

Reply:
Thanks a lot @atymic 👍